### PR TITLE
feat: add command clusteradm get placements

### PR DESCRIPTION
**Pull request by ycyaoxdu**
Signed-off-by: ycyaoxdu <yaoyuchen0626@163.com>

### CODE REVIEW

**Comment by ycyaoxdu** [comment]
example output:
```
root@yuchen clusteradm [feat] $ clusteradm get placements
NAME                   STATUS   REASON                  SELETEDCLUSTERS
placement-policy-pod   True     AllDecisionsScheduled   1
```

**Comment by ycyaoxdu** [comment]
#133  #163

**Comment by qiujian16** [comment]
we might want to show more details, such as the clusters in the related decisions

**Comment by ycyaoxdu** [comment]
use a slice of cluster names to replace the number
```
NAME                               STATUS       REASON                            SELETEDCLUSTERS
placement-policy-pod   True              AllDecisionsScheduled    [cluster1]
```

**Comment by ycyaoxdu** [comment]
```
➜  clusteradm git:(feat/get-placement) ✗ clusteradm get placements
<Placement>
└── placement1
    └── <Namespace>: default
    └── <ClusterSet>: [default]
    └── <Status>
    │   ├── <Conditions>
    │   │   ├── PlacementSatisfied -> true
    │   │   ├── PlacementMisconfigured -> unknown
    │   ├── <Number of Selected Clusters>: 2
    └── <PlacementDecision>
        └── cluster1
        └── cluster2
➜  clusteradm git:(feat/get-placement) ✗ clusteradm get placements -o table
NAMESPACE   NAME         STATUS   REASON                  SELETEDCLUSTERS
default     placement1   True     AllDecisionsScheduled   [cluster1 cluster2]
```

@qiujian16 PTAL

**Comment by qiujian16** [comment]
should we print table by default or tree by default? I think we are not quite consistent on other commands.

**Comment by ycyaoxdu** [comment]
> should we print table by default or tree by default? I think we are not quite consistent on other commands.

@qiujian16  maybe we should use tree as the default get output format? some command can not print as table, like `hub-info`, `klusterlet-info` , `token`.

**Comment by qiujian16** [comment]
some may not have tree format, like `get clusters`? Or we should also change get clusters default format

**Comment by ycyaoxdu** [comment]
> should we print table by default or tree by default? I think we are not quite consistent on other commands.

currently i've set tree to default. i will

> 

ok, will do this in https://github.com/open-cluster-management-io/clusteradm/pull/174

File: pkg/cmd/get/placement/exec.go [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+)
+
+const placementLabel = "cluster.open-cluster-management.io/placement"
+
+func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
+	return nil
+}
+
+func (o *Options) validate(args []string) (err error) {
+	if len(args) > 0 {
```
Comment (qiujian16) [inline pkg/cmd/get/placement/exec.go:28]: nit, we should allow input name of the placement. And then you could use list with fieldSelector.

**Pull request merged**